src/main/webapp/resources/js/components/samples/components/SampleFileContenate.tsx

```diff
@@ -49,26 +52,33 @@ export function SampleFileConcatenate({
     (state) => state.sampleFilesReducer
   );
   const [concatenateSeqObjectFiles] = useConcatenateSequencingObjectsMutation();
+  const [concatenateButtonDisabled, setConcatenateButtonDisabled] =
+    React.useState<boolean>(true);
+  const [concatenating, setConcatenating] = React.useState<boolean>(false);
+  const [concatenationError, setConcatenationError] =
+    React.useState<string>("");
 
   useResetFormOnCloseModal({
     form,
     visible,
   });
 
   const concatenateFiles = () => {
+    setConcatenationError("");
     const sequencingObjectIds = concatenateSelected.map(
       (seqObject: SequencingObject) => seqObject.identifier
     );
 
     form.validateFields().then((values) => {
+      setConcatenating(true);
       concatenateSeqObjectFiles({
         sampleId: sample.identifier,
         sequencingObjectIds: sequencingObjectIds,
         newFileName: values.new_file_name,
         removeOriginals: values.remove_original_files,
       })
         .unwrap()
-        .then((data: SampleSequencingObject[]) => {
+        .then(({ sampleSequencingObjectFileModels }) => {
           let message = i18n("SampleFilesConcatenate.concatenationSuccess");
```

Comment:
These strings should probably be in the response

Reply:
Updated in 2c94eb7d49c94c036f27427d7d02746f56a089d3